### PR TITLE
Fix incorrect links in workload overview page

### LINKED ISSUE #5932: Some workload overview links navigate to deployments tab
https://user-images.githubusercontent.com/40840436/181382733-f331d573-e71f-4777-9670-eefbdce7dcfd.mov

Stateful Sets, Replica Sets and Cron Jobs

6.0.0-alpha

### PR DESCRIPTION
**Pull request by jansav**
Fixes #5932 

Signed-off-by: Janne Savolainen <janne.savolainen@live.fi>

### CODE REVIEW

**Review by aleksfront — APPROVED** [review]
LGTM

File: src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts [hunk 1]
```diff
@@ -5,16 +5,16 @@
 import { getInjectable } from "@ogre-tools/injectable";
 import { workloadInjectionToken } from "../workload-injection-token";
 import { ResourceNames } from "../../../../utils/rbac";
-import navigateToPodsInjectable from "../../../../../common/front-end-routing/routes/cluster/workloads/deployments/navigate-to-deployments.injectable";
```
Comment (jim-docker) [inline src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts:8]: the downside of default exports? `navigateToPodsInjectable` from `navigate-to-deployments.injectable`? 

This looks like two levels of copy-paste errors :)

**Review by jim-docker — APPROVED** [review]

File: src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts (hunk 1 above)
Comment (jansav) [reply src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts:8]: I agree with you on this. I didn't even notice the wrong name, and was thinking "how this lead to deployments.."

Comment (jansav) [reply src/renderer/components/+workloads-overview/workloads/implementations/statefulsets-workload.injectable.ts:8]: But of course, there should be unit test to cover it. :) Sooo, I'd say that problem is elsewhere.

**Pull request merged**